coordinator/src/node/connection.rs

```diff
@@ -28,7 +28,10 @@ pub async fn keep_public_channel_peers_connected(
     }
 }
 
-fn reconnect_to_disconnected_public_channel_peers(node: Arc<Node<InMemoryStore>>) {
+fn reconnect_to_disconnected_public_channel_peers<S>(node: Arc<Node<S>>)
+where
+    S: Storage + Sync + Send + 'static,
```

Comment:
🔧 I'm going for a concrete type here. I don't think we need to abstract over this type (in this crate at least).

Reply:
Makes sense! I only did this in case we want to re-use these functions in e.g. `test`, but given that this is in the executable that won't be necessary. Please use the concreate type when you rebase :)